elements/pfe-tabs/demo/pfe-tabs.html

```diff
@@ -450,7 +450,7 @@ <h2>Content 2</h2>
           <p>Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est
             laborum.</p>
         </pfe-tab-panel>
-        <pfe-tab role="heading" slot="tab" id="tab3">Tab 3</pfe-tab>
+        <pfe-tab role="heading" slot="tab" id="tab3-a">Tab 3</pfe-tab>
```

Comment:
Was the -a added here for a test before?

Reply:
🤔 can't remember. It's probably because there's a duplicate id on the page